src/evm_context.rs

```diff
@@ -22,6 +23,13 @@ pub fn init(ssa: &mut SSATracker) -> String {
             .join("")
             .as_str(),
     );
+
+    let _ = write!(
```

Comment:
Is this really the best way of doing this?

Reply:
That's what clippy suggests, unfortunately.